features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/tags_api_router.jag

```diff
@@ -65,6 +65,7 @@ routeManager.register('GET', 'publisher', '/publisher/api/tag/{type}/{id}', func
 
     for (var index in tags) {
         tagArray.push({id: counter, name: tags[index]});
+        counter++;
```

Comment:
Since this is a for-each loop, in each iteration `index` variables is a 'tag', not a integer.